demo/demo.js

```diff
@@ -58,27 +69,27 @@ const showTooltip = commit => {
                     </p>
                 </div>
             </div>
-        `)
-        .classed(direction, true)
-        .style({
-            left: `${left}px`,
-            top: (d3.event.pageY + 16) + 'px',
-        });
+        `);
+        tooltip.style('left', `${left}px`)
```

Comment:
Doesn't D3 4 supports an object as a style parameter? I prefer this kind of writing for readability.